backend/src/main/java/gov/cdc/usds/simplereport/db/model/TestOrder.java

```diff
@@ -127,7 +133,19 @@ public Boolean getNoSymptoms() {
 		return askOnEntrySurvey.getSurvey().getNoSymptoms();
 	}
 
+	@Override
```

Comment:
This shouldn't be an override, though—superclass is explicitly not supposed to have this method